server/template.js

```diff
@@ -19,6 +28,23 @@ export default ({ url, bodyClass, title, description }) => marinate`
   <meta name="theme-color" content="${ONLINE_COLOR}">
   <meta name="description" content="${description}">
   <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="manifest" href="/manifest.json">
+
+  <!-- Preconnect -->
+  <link rel="preconnect" href="//cdn.polyfill.io" crossorigin>
+  <link rel="preconnect" href="//api.banidb.com" crossorigin>
+
+  <!-- Preload Fonts -->
+  <link async rel="preload" href="/assets/fonts/GurbaniAkharHeavyTrue.ttf?v=1" as="font">
+  <link async rel="preload" href="/assets/fonts/AnmolLipiSG.ttf?v=1" as="font">
+
+  <!-- Preload JavaScript -->
+  <link async rel="preload" href="${manifest['Search.js']}" as="script">
+  <link async rel="preload" href="${manifest['Shabad.js']}" as="script">
+
+  <!-- Preload Images -->
+  <link async rel="preload" href="/assets/images/sttm_logo.png" as="image">
+  <link async rel="preload" href="/assets/images/logo-192x192.png" as="image">
```

Comment:
Bad merge? Dev has automated strings for this.

Reply:
Remove all these lines as well. Check the *HTML variables, they're handling these now.